app/src/main/java/com/github/scribeWizTeam/scribewiz/fragments/NotesListFragment.kt

```diff
@@ -107,14 +107,25 @@ class NotesListFragment(contentLayoutId: Int) : Fragment(contentLayoutId) {
                             ShareMenu(sharedNoteName.value, showShareMenu)
                         }
 
```

Comment:
This is a great idea to warn of the users if they have no notes and to null check!